src/middlewares/validation/validation.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
```

Comment:
why disable camelCase?

Reply:
According to what the TTL told me. Move the validation folder into the helpers folder.

Reply:
Alright, it has be done